fail2ban/server/filter.py

```diff
@@ -38,6 +38,11 @@
 from .action import CommandAction
 from ..helpers import getLogger
 
+try:
+	from collections import OrderedDict
+except ImportError:
+	OrderedDict = dict
+
```

Comment:
http://nbviewer.ipython.org/url/www.onerussian.com/tmp/dics-vs-ordereddict.ipynb
so why should we use slower OrderedDict?